common/notify-failed-test

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+export COMMON=${COMMON:-$(cd $(dirname $BASH_SOURCE); echo $PWD)}
+export SESCI=${SESCI:-$(cd $(dirname $BASH_SOURCE)/..; echo $PWD)}
+
+source $COMMON/util
+
+export JUNIT=${JUNIT:-"res/make-check.xml"}
+# space separated
+export NOTIFY_FAILED_TEST_CHANNELS=${NOTIFY_FAILED_TEST_CHANNELS:-""}
+# newline separated
+export NOTIFY_FAILED_TEST_LIST=${NOTIFY_FAILED_TEST_LIST:-""}
+
+export FAILED_LIST=${FAILED_LIST:-""}
+
+if [ -z "$FAILED_LIST" ] ; then
+    echo "No failed test detected"
+    exit 1
+fi
+echo "Following tests were found among failed: $FAILED_LIST"
+echo "Test environment will not be destroyed"
+
+if [ ! -z "$STATUS" ] ; then
+    export SERVER_NAME=${SERVER_NAME:-"$(cat $STATUS | yj | jq -r .server.name)"}
+    export SERVER_ADDR=${SERVER_ADDR:-"$(cat $STATUS | yj | jq -r .server.ip)"}
+else
+    export SERVER_NAME=${SERVER_NAME:-"unknown-hostname"}
+    export SERVER_ADDR=${SERVER_ADDR:-"1.2.3.4"}
+fi
+
+tee message.txt << EOF
+The jenkins job $JOB_NAME build $BUILD_ID is failed. Following tests found among failed:
+
+$(for i in $FAILED_LIST ; do echo "[$i]($BUILD_URL/testReport/%28root%29/make-check/$i/)" ; done)
+
+The make check node '$SERVER_NAME' can be reached by ip '$SERVER_ADDR':
+
+\`\`\`
+ssh -i ~/.ssh/sa root@$SERVER_ADDR
+\`\`\`
+
+The node can be destroyed by the command:
+
+\`\`\`
+openstack --os-cloud '$TARGET_CLOUD' server delete '$SERVER_NAME'
```

Comment:
link to the manual about our openstack usage, would be cool.

Reply:
Not sure we have one. Please give the link if you have something in mind?

Reply:
I can't find it as well. But I think that will be first question from dev.

Reply:
Well... it is not our openstack, it is located on OVH right now, but it can be switched to ECP at some point. I suppose a developer can reach out his manager to ask for access to the openstack.